test/plugins/relativeTime.test.ts

```diff
@@ -0,0 +1,70 @@
+import { esday } from 'esday'
+import moment from 'moment/min/moment-with-locales'
+import { describe, expect, it } from 'vitest'
+import relativeTimePlugin from '~/plugins/relativeTime'
+
+esday.extend(relativeTimePlugin)
+
+describe('relativeTime plugin', () => {
+  it.each([
+    { input: esday().subtract(30, 'second') },
+    { input: esday().subtract(1, 'minute') },
+    { input: esday().subtract(5, 'minute') },
+    { input: esday().subtract(1, 'hour') },
+    { input: esday().subtract(3, 'hour') },
+    { input: esday().subtract(1, 'day') },
+    { input: esday().subtract(10, 'day') },
+    { input: esday().add(30, 'second') },
+    { input: esday().add(1, 'minute') },
+    { input: esday().add(5, 'day') },
+    { input: esday().add(1, 'year') },
+  ])('fromNow() output should match moment.fromNow()', ({ input }) => {
+    const esResult = input.fromNow()
+    const momentResult = moment(input.toDate()).fromNow()
+
+    expect(esResult).toBe(momentResult)
+  })
+
+  it('toNow() should match moment.toNow()', () => {
+    const input = esday().add(3, 'hour')
+    const esResult = esday().to(input)
+    const momentResult = moment().to(moment(input.toDate()))
+
+    expect(esResult).toBe(momentResult)
```

Comment:
Perhaps using `expectSame` as mentioned above would make this test more readable.